lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,13 +1,26 @@
-<img class="header-img" src="<%- image_url %>" alt="<%- display_name %>"/>
-<div class="course-details">
-    <h3 class="course-title"><%- display_name %></h3>
-    <p class="course-key"><%- key %></p>
-    <% if(enrollment_status){ %>
-        <span class="enrollment-info"><%- course_start %> - <%-course_end %></span>
-        <a href="<%- courseware_url %>" class="course-link cta"><%- gettext('View Course') %></a>
-    <% }else{ %>
-        <span class="enrollment-info"><%- gettext('Not Yet Enrolled') %></span>
-        <a href="<%- marketing_url %>" class="course-link cta"><%- gettext('Learn More') %></a>
-    <% } %>
+<div class="grid-container grid-manual">
+    <div class="course-meta-container col-12 xl-col-8 lg-col-8 md-col-8 sm-col-12">
+        <a href="<%- marketing_url %>" class="course-image-link">
+            <img
+                class="header-img"
+                src="<%- course_image_url %>"
+                alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true)%>"/>
+        </a>
+        <div class="course-details">
+            <h3 class="course-title">
+                <a href="<%- marketing_url %>" class="course-title-link">
+                    <%- display_name %>
+                </a>
+            </h3>
+            <div class="course-text">
+                <% if(is_enrolled){ %>
```

Comment:
Nitpick: there should be a space between ```if``` and ```(is_enrolled)``` to make it explicit that this is not a function.